grafana-plugin/src/containers/UserSettings/parts/tabs/MSTeamsInfo/MSTeamsInfo.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+
+import cn from 'classnames/bind';
+import { observer } from 'mobx-react';
+
+import MSTeamsInstructions from 'components/MSTeams/MSTeamsInstructions';
+import Text from 'components/Text/Text';
+import { useStore } from 'state/useStore';
+
+import styles from 'containers/UserSettings/parts/tabs/MSTeamsInfo/MSTeamsInfo.module.css';
+
+const cx = cn.bind(styles);
+
+const MSTeamsInfo = observer(() => {
+  const store = useStore();
+  const { userStore, msteamsChannelStore } = store;
```

Comment:
We could destructure in one line `const { userStore, msteamsChannelStore } = useStore();`